src/navs/documentation.js

```diff
@@ -91,7 +91,7 @@ export const documentationNav = {
     pages['series-series'],
   ],
   String: [
-    pages['string-camal'],
+    pages['string-camel'],
```

Comment:
Hey @danish-mehmood I think the build is failing because the page `string-camel` doesn't exist. Just renaming that file to the correct spelling to match should fix 👍

Reply:
ok i will fix it